articles/quickstart/webapp/aspnet-core/_includes/_login.md

```diff
@@ -237,13 +248,9 @@ public void ConfigureServices(IServiceCollection services)
 }
 ```
 
-This will ensure that when `SignOutAsync` is called for the OIDC Middleware, that the `/v2/logout` endpoint of the Auth0 Authentication API is called to log the user out of Auth0. 
-
-It will also pass along the Redirect URL (when specified) in the `returnTo` parameter. You must therefore ensure that you have specified this URL in the **Allowed Logout URLs** for your Client in the Auth0 Dashboard.
-
-## Add Login and Logout Links
+## Add the Log In and Log Out Buttons
 
-Lastly, add Login and Logout links to the navigation bar. To do that, head over to `/Views/Shared/_Layout.cshtml` and add code to the navigation bar section which displays a Logout link when the user is authenticated, otherwise a Login link. These will link to the `Logout` and `Login` actions of the `AccountController` respectively:  
+Add the **Log In** and **Log Out** buttons to the navigation bar. In the `/Views/Shared/_Layout.cshtml` file, in the navigation bar section, add code that displays the **Log Out** button when the user is authenticated and the **Log In** button in any other case. The buttons link to the `Logout` and `Login` actions in the `AccountController`:  
```

Comment:
> ...and the **Log In** button in any other case.

There are only 2 cases: either authenticated or not. So I think rather change this to 

> and the **Log In** button if not.